base/initdefs.jl

```diff
@@ -235,7 +235,8 @@ function init_active_project()
     ACTIVE_PROJECT[] =
         project === nothing ? nothing :
         project == "" ? nothing :
-        project == "@." ? current_project() : abspath(expanduser(project))
+        project == "@." ? current_project() :
+        startswith(project, "@") ? abspath(load_path_expand(project)) : abspath(expanduser(project))
```

Comment:
```suggestion
        startswith(project, "@") ? load_path_expand(project) : abspath(expanduser(project))
```

Looking at `load_path_expand`, it already calls `current_project` and `abspath`, and might return `nothing`